lib/Twig/Extension/Core.php

```diff
@@ -1264,6 +1264,10 @@ function _twig_escape_html_attr_callback($matches)
      */
     function twig_length_filter(Twig_Environment $env, $thing)
     {
+        if (null === $thing) {
```

Comment:
If the var passed to the filter is dynamic and not set at the moment, this may cause ```Undefined variable``` PHP notice I guess. Probably ```isset()``` would be safer.